stairctl/build.gradle

```diff
@@ -0,0 +1,106 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+}
+
+plugins {
+    id 'java'
+    id 'application'
+    id 'com.github.spotbugs' version '4.0.8'
+    id "io.spring.dependency-management" version "1.0.10.RELEASE"
+    id "org.springframework.boot" version "2.4.0"
+    // Spotless doesn't work still
```

Comment:
optional  nit: consider filing a ticket for tracking, or just remove the commented out code.

Reply:
PF-867